products/zentyal.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Zentyal
+category: os
+tags: linux-distribution
+permalink: /zentyal
+releasePolicyLink: https://zentyal.com/release-policy/
+releaseDateColumn: true
```

Comment:
```suggestion
releaseDateColumn: true
releaseColumnLabel: false
```

There seem to be not reliable source to obtain the latest version for each release with their date. I think it's better to drop this column.

Reply:
Done in https://github.com/endoflife-date/endoflife.date/pull/5182/commits/aef3c0c1480d8280dd4dc58ec9da43de35610495.